driver-embedded/src/main/com/mongodb/embedded/client/EmbeddedDatabase.java

```diff
@@ -61,15 +60,8 @@ public void close() {
         }
     }
 
-    private String[] createCmdArgs(final MongoClientSettings mongoClientSettings) {
-        return asList(
-                "--port=0",
-                "--storageEngine=mobile",
-                "--dbpath=" + mongoClientSettings.getDbPath()
-        ).toArray(new String[0]);
+    private String createYamlConfig(final MongoClientSettings mongoClientSettings) {
+        return format("{ storage: { dbPath: %s } }", mongoClientSettings.getDbPath());
```

Comment:
Configurations for db_new have deliberately been kept simple for now, we may need to review this in the future.